web-js/buffer-geocoder-result/index.html

```diff
@@ -0,0 +1,208 @@
+        var nullSymbol = new SimpleMarkerSymbol().setSize(0);
+        featureLayer.setRenderer(new SimpleRenderer(nullSymbol));
+        
+        map.addLayer(featureLayer);
+        
+        locator = new Locator("http://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer");
+        locator.on("address-to-locations-complete", showResults);
+
+        // listen for button click then geocode
+        registry.byId("locate").on("click", locate);
+        
+        map.infoWindow.resize(200,125);
+
+        function locate() {
+          map.graphics.clear();
+          var address = {
+            "SingleLine": dom.byId("address").value
+          };
+          locator.outSpatialReference = map.spatialReference;
+          var options = {
+            address: address,
+            outFields: ["Loc_name"]
+          };
+          locator.addressToLocations(options);
+        }
+		
+		var circleSymb = new SimpleFillSymbol(
+          SimpleFillSymbol.STYLE_NULL,
+          new SimpleLineSymbol(
+            SimpleLineSymbol.STYLE_SHORTDASHDOTDOT,
+            new Color([105, 105, 105]),
+            2
+          ), new Color([255, 255, 0, 0.25])
+        );
+        var circle;
+
+        function showResults(evt) {
+          var symbol = new SimpleMarkerSymbol();
+          var infoTemplate = new InfoTemplate(
+            "Location", 
+            "Address: ${address}<br />Score: ${score}<br />Source locator: ${locatorName}"
+          );
+          symbol.setStyle(SimpleMarkerSymbol.STYLE_SQUARE);
+          symbol.setColor(new Color([153,0,51,0.75]));
+
+          var geom;
+          arrayUtils.every(evt.addresses, function(candidate) {
+            console.log(candidate.score);
+            if (candidate.score > 80) {
+              console.log(candidate.location);
+              var attributes = { 
+                address: candidate.address, 
+                score: candidate.score, 
+                locatorName: candidate.attributes.Loc_name 
+              };   
+              geom = candidate.location;
+              var graphicLoc = new Graphic(geom, symbol, attributes, infoTemplate);
+              //add a graphic to the map at the geocoded location
+              map.graphics.add(graphicLoc);
+			  
+			  circle = new Circle({
+				center: geom,
+				geodesic: true,
+				radius: 1000,
+				radiusUnit: "esriMeters"
+				});
+			  
+			  var graphic = new Graphic(circle, circleSymb);
+			  map.graphics.add(graphic);
+
+              var query = new Query();
+              query.geometry = circle.getExtent();
+              //use a fast bounding box query. will only go to the server if bounding box is outside of the visible map
+              featureLayer.queryFeatures(query, selectInBuffer);
+			  
+              return false; //break out of loop after one candidate with score greater  than 80 is found.
+            }
+          });
+          if ( geom !== undefined ) {
+            map.centerAndZoom(geom, 14);
+		  }
```

Comment:
this is nitpicky, but your tabbing/formatting is a little wonky in places.

also, it looks like you are querying using the boundingBox extent of the Circle geometry (as opposed to the circle itself).  i don't think thats a problem, but probably worth mentioning in the readme.